shortlister/tournament.py

```diff
@@ -0,0 +1,142 @@
+from copy import deepcopy
+from itertools import combinations
+from pathlib import Path
+import pickle
+from typing import Dict, List
+
+
+def comparison(mylist, result):
+    """Starts the comparison process"""
+    pairs = get_pair(
+        mylist, result
+    )  # returns a list of pairs that can be used to compare
+    for pair in pairs:
+        print(f"{pair[0].name}[l] : {pair[1].name}[r]")
+        winner = choose(pair)
```

Comment:
I wonder...would it be possible to undo the most recent choice, somehow?

Reply:
Maybe removing the lastest pair from the results dictionary would work?

Reply:
Yes, but you'd also need to place it back into the pairs list so it gets evaluated again. Not sure how the user interaction would work...go to the next pair, if user clicks 'u' it's undo the previous pair? Play around with some ideas to see what works best.